Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -81,7 +81,22 @@ private func header(_ lane: Pricing.Lane) -> [Node] {
   ]
 }
 
-private func teamMembers(_ currentUser: User, _ subscribeData: SubscribeConfirmationData) -> [Node] {
+private func teamMembers(
+  lane: Pricing.Lane,
+  currentUser: User,
+  subscribeData: SubscribeConfirmationData
+  ) -> [Node] {
+
+  guard lane == .team else {
+    return [
+      input([
+        name(SubscribeData.CodingKeys.isOwnerTakingSeat.rawValue),
+        `type`(.hidden),
+        value("true")
+        ])
```

Comment:
I need to include this hidden input for individual subscriptions so that it gets properly set in the `SubscribeData` posted to the server.